react/features/invite/components/add-people-dialog/web/InviteContactsForm.tsx

```diff
@@ -1,63 +1,92 @@
-// @flow
-
-import InlineMessage from '@atlaskit/inline-message';
+import { withStyles } from '@mui/styles';
 import React from 'react';
 import { connect } from 'react-redux';
-import type { Dispatch } from 'redux';
 
+import { IReduxState, IStore } from '../../../../app/types';
 import Avatar from '../../../../base/avatar/components/Avatar';
 import { translate } from '../../../../base/i18n/functions';
 import Icon from '../../../../base/icons/components/Icon';
 import { IconPhoneRinging } from '../../../../base/icons/svg';
 import MultiSelectAutocomplete from '../../../../base/react/components/web/MultiSelectAutocomplete';
+import Button from '../../../../base/ui/components/web/Button';
+import { BUTTON_TYPES } from '../../../../base/ui/constants.any';
 import { isVpaasMeeting } from '../../../../jaas/functions';
-import { hideAddPeopleDialog } from '../../../actions';
+import { hideAddPeopleDialog } from '../../../actions.web';
 import { INVITE_TYPES } from '../../../constants';
+import { IInvitee, InviteSelectItem } from '../../../types';
 import AbstractAddPeopleDialog, {
     type Props as AbstractProps,
     type State,
     _mapStateToProps as _abstractMapStateToProps
 } from '../AbstractAddPeopleDialog';
 
-declare var interfaceConfig: Object;
+const styles = () => {
+    return {
+        formWrap: {
+            marginTop: '8px'
+        },
+        inviteButtons: {
+            display: 'flex',
+            justifyContent: 'end',
+            marginTop: '8px',
+            '&>button': {
```

Comment:
Could we use some class instead please? Tag selectors are not the best for performance